app/src/main/java/com/cornellappdev/score/components/EmptyStateMessage.kt

```diff
@@ -0,0 +1,59 @@
+package com.cornellappdev.score.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.score.theme.GrayLight
+import com.cornellappdev.score.theme.GrayMedium
+import com.cornellappdev.score.theme.GrayPrimary
+import com.cornellappdev.score.theme.Style.body
+import com.cornellappdev.score.theme.Style.heading2
+import com.example.score.R
+
+@Preview
+@Composable
+fun PreviewEmptyStateMessage() {
+    Box(modifier = Modifier.background(Color.White)) {
+        EmptyStateMessage()
+    }
+}
+
+@Composable
+fun EmptyStateMessage() {
+    Column(modifier = Modifier.width(158.dp),
```

Comment:
To ensure we can align this thing and whatnot in a screen later, let's take in a `modifier: Modifier = Modifier` and give it to the `Column` (i.e. `Column(modifier = modifier.width(158.dp),`)

Reply:
That way if we need to pass in some `Modifier.align()`, for example, we can do that